autoload/fzf_tags.vim

```diff
@@ -9,6 +9,8 @@ let s:actions = {
   \ 'ctrl-x': 'split',
   \ 'ctrl-v': 'vsplit' }
 
+let s:default_layout = { 'down': '~40%' }
```

Comment:
Hmm, the more I think about it, the more I think we should get out of the way and let the normal FZF configuration do its thing.

The FZF experience should be consistent across all the tools that use it. How about we remove the default layout altogether, and in the README we point the user to the global FZF configurations?

https://github.com/junegunn/fzf/blob/master/README-VIM.md#configuration

Would this solve your layout customization need?

Reply:
@zackhsi +1 for the default FZF look. I've just spent 20 minutes to find and remove this `~40%` thing (i'm just a casual fzf user, so bear with me :)

Reply:
@funbringer I've addressed this in master by using the pop-up layout as default, and supporting the user's g:fzf_layout setting.

See:
- https://github.com/zackhsi/fzf-tags/issues/13
- https://github.com/zackhsi/fzf-tags/pull/14